.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 .vscode/
 .idea/
 third_party/shellcheck/download/
+macos-signing-certificate.p12
```

Comment:
Why is this? Delete?

Reply:
We have to pull down this file from our internal storage and place it here for it to work, I added it to the gitignore to avoid anyone accidentally committing it.